README.md

```diff
@@ -1,7 +1,13 @@
 # What is this?
 
+The github.dev web-based editor is a lightweight editing experience that runs entirely in your browser. You can navigate files and source code repositories from GitHub, and make and commit code changes.
+
 Press the `.` key on any repository or pull request, or swap `.com` with `.dev` in the URL, to go directly to a VS Code environment in your browser.
 
+To get started coding and adding files, try out the `.` key on any of your repos or PRs. You can also check out [http://github.dev/microsoft/vscode](http://github.dev/microsoft/vscode) as an example of github.dev in action. 
```

Comment:
This and line 5 are pretty similar. Could we just expand both of them like: 

There are two ways to go directly to a VS Code environment in your browser and start coding:
- Press the `.` key on any repository or pull request.
- Swap `.com` with `.dev` in the URL. For example, for this repo https://github.com/github/dev becomes http://github.dev/github/dev

I'm also wondering if most people are seeing this readme in the github.dev environment. In which case we probably don't want to confuse them too much by linking to another github.dev repo.

Reply:
Thanks @timeyoutakeit! Your feedback makes sense. I updated with the wording you suggested.

Reply:
it works, Yesss!

Reply:
Tip feature Functional, Thanx!